ports/stm32/boards/MIKROE_QUAIL/mpconfigboard.h

```diff
@@ -0,0 +1,113 @@
+#define MICROPY_HW_BOARD_NAME       "MikroE Quail"
+#define MICROPY_HW_MCU_NAME         "STM32F427VI"
+#define MICROPY_PY_SYS_PLATFORM     "MikroE Quail"
```

Comment:
This line should probably be removed so the setting stays as the default of `pyboard`.

Reply:
Just the last one or all of them? It'd be nice to see the first two board references when running `mpremote`.

Reply:
Just the last one.  You'll still see the banner with the full board name and MCU.

Reply:
OK.